class3.2/include/test.cpp

```diff
@@ -0,0 +1,76 @@
+#include "UnitTests.hpp"
+#include <cmath>
+#include <gtest/gtest.h> 
+
+class Triangle {
+public:
+    Triangle(double a, double b, double c) : side1(a), side2(b), side3(c) {}
+
+    double area() const {
+        double s = (side1 + side2 + side3) / 2.0;
+        return std::sqrt(s * (s - side1) * (s - side2) * (s - side3));
+    }
+
+private:
+    double side1, side2, side3;
+};
+
+void test_triangle_area() {
+    // Test cases for triangle area
+    Triangle triangle1(3, 4, 5);
+    ASSERT_EQ(triangle1.area(), 6.0);
```

Comment:
it is better to split in few test cases.
But there is another question - we still have a possibility to create failed test. What about 0 or negative values for a,b, or c. Another example does any triple of positive numbers a,b,c form a triangle?